### ISSUE TITLE
Accessibility issue with ProxyClassAccessibility

### ISSUE THREAD

**Issue by FatSwords** [StefH/ProxyInterfaceSourceGenerator#59]
Hi,

It looks like the generated classes use ProxyClassAccessibility which is internal, however their methods are public, which gives a compilation error.

I think here:

https://github.com/StefH/ProxyInterfaceSourceGenerator/blob/994e55c4be38f1b89b1c5735e1c3e10621f6c826/src/ProxyInterfaceSourceGenerator/FileGenerators/ProxyAttributeGenerator.cs#L52

needs to become 

`public enum ProxyClassAccessibility`

**Comment by StefH** [comment]
I think you are correct. I will create a new version.

**Comment by StefH** [comment]
@FatSwords 
After some thinking, the issue is not the ProxyClassAccessibility, that enum should be internal.

The problem was related to internal sealed class ProxyAttribute : Attribute

A new NuGet is released.
